sklearn/gaussian_process/gpc.py

```diff
@@ -189,8 +189,9 @@ def fit(self, X, y):
                              "y contains classes %s"
                              % (self.__class__.__name__, self.classes_))
         elif self.classes_.size == 1:
-            raise ValueError("{0:s} requires 2 classes.".format(
-                self.__class__.__name__))
+            raise ValueError("{0:s} requires 2 classes; got {1:d} class"
```

Comment:
If this PR is test-driven, why does this appear to have no test coverage?

Reply:
That's because I must have changed all error messages in the file gpc.py related to the fact that there is only one class. However this is the error message of the estimator `_BinaryGaussianProcessClassifierLaplace` which is ignored in the common tests because of the name starts with `_`.